cpp/log/cert_checker.cc

```diff
@@ -292,26 +293,27 @@ Status CertChecker::GetTrustedCa(CertChain* chain) const {
        it != issuer_range.second; ++it) {
     const Cert* issuer_cand = it->second;
 
-    Cert::Status ok = subject->IsSignedBy(*issuer_cand);
-    if (ok == Cert::UNSUPPORTED_ALGORITHM) {
+    StatusOr<bool> signed_by_issuer = subject->IsSignedBy(*issuer_cand);
+    if (signed_by_issuer.status().CanonicalCode() == Code::UNIMPLEMENTED) {
```

Comment:
I wonder if we shouldn't just pass back the underlying status, or if in fact, this sort of translation is good (to control better what the end user see in the HTTP responses)?